board/freescale/imx8mm_evk/spl.c

```diff
@@ -282,3 +282,32 @@ int do_reset(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 
 	return 0;
 }
+
+#ifdef CONFIG_SPL_MMC_SUPPORT && SECONDARY_BOOT_RUNTIME_DETECTION
+unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
```

Comment:
This doesn't have the same function signature as here (where it is called):
https://github.com/foundriesio/u-boot/blob/2020.04%2Bfio/common/spl/spl_mmc.c#L415

```
		raw_sect = spl_mmc_get_uboot_raw_sector(mmc, raw_sect);
```

Reply:
just borrowed that from another board file (`imx8mn_evk`):
```
u-boot-fio$ grep -e spl_mmc_get_uboot_raw_sector -r ./
./arch/arm/mach-sunxi/board.c:unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
./arch/arm/mach-imx/imx8/image.c:unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc,
./common/spl/spl_mmc.c:unsigned long __weak spl_mmc_get_uboot_raw_sector(struct mmc *mmc,
./common/spl/spl_mmc.c:		raw_sect = spl_mmc_get_uboot_raw_sector(mmc, raw_sect);
./board/freescale/imx8mn_evk/spl.c:unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
./board/freescale/imx8mp_evk/imx8mp_evk.c:unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
./board/toradex/verdin-imx8mp/verdin-imx8mp.c:unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc)
...
```
Interesting that compiler doesn't catch that and mark as error.